google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/project/DefaultAppEngineProjectService.java

```diff
@@ -321,23 +321,24 @@ String getServiceNameFromAppEngineWebXml(XmlFile appengineWebXml) {
   }
 
   @Override
-  public void generateAppYaml(FlexibleRuntime runtime, Module module) {
+  public void generateAppYaml(FlexibleRuntime runtime, Module module, Path outputFolderPath) {
     Properties templateProperties =
         FileTemplateManager.getDefaultInstance().getDefaultProperties();
     templateProperties.put("RUNTIME", runtime.toString());
 
     ApplicationManager.getApplication().runWriteAction(() -> {
       generateFromTemplate(
           AppEngineTemplateGroupDescriptorFactory.APP_YAML_TEMPLATE,
-          "appengine",
           "app.yaml",
```

Comment:
If you really want to keep the config Enum, you could use it here to avoid hardcoding "app.yaml"